ui/lib/core/addon/components/navigate-input.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="navigate-filter">
+  <div class="field" data-test-nav-input>
+    <p class="control has-icons-left">
+      {{! template-lint-disable no-down-event-binding }}
+      <input
+        class="filter input"
+        value={{@filter}}
+        placeholder={{or @placeholder "Filter keys"}}
+        type="text"
+        data-test-comoponent="navigate-input"
+        {{on "input" this.handleInput}}
+        {{on "keyup" this.handleKeyUp}}
+        {{on "keydown" this.handleKeyPress}}
+        {{on "focus" (fn this.setFilterFocused true)}}
+        {{on "blur" (fn this.setFilterFocused false)}}
+        {{! template-lint-disable no-autofocus-attribute}}
+        autofocus={{this.focusFilter}}
```

Comment:
We do autofocus in several places. While I can see why this is bad practice, I think a separate PR addressing this usage through the app should address this. Right now, if I remove this it does mess with the current functionality.

Reply:
(needed because I removed the extend FocusOnInsertMixin from the component's js.

Reply:
Removed. Good call.